packages/app/src/api.ts

```diff
@@ -531,7 +531,8 @@ const api = {
       {
         name: string;
         query: string;
-        tags?: string;
+        tags?: string[];
```

Comment:
@MikeShi42 Changed tag type from string to string[], not sure if that is accurate. Please correct if wrong